src/main/java/io/jenkins/plugins/orka/helpers/FormValidator.java

```diff
@@ -29,8 +31,8 @@ public FormValidation doCheckConfigName(String configName, String orkaEndpoint,
 
             try {
                 if (StringUtils.isNotBlank(orkaEndpoint) && orkaCredentialsId != null) {
-                    OrkaClientProxy clientProxy = this.clientProxyFactory
-                        .getOrkaClientProxy(orkaEndpoint, orkaCredentialsId);
+                    OrkaClientProxy clientProxy = this.clientProxyFactory.getOrkaClientProxy(orkaEndpoint,
+                            orkaCredentialsId);
```

Comment:
Not needed space before `orkaCredentialsId ` ? It seems we need linting rules :D

Reply:
We have :) but we should update them :D

Reply:
And we'll need to run them on CI build. But it is an another task :)